lib/mosaic-database/src/main/java/org/eclipse/mosaic/lib/database/persistence/SQLiteLoader.java

```diff
@@ -16,6 +16,8 @@
 package org.eclipse.mosaic.lib.database.persistence;
 
 import org.eclipse.mosaic.lib.database.Database;
+import org.eclipse.mosaic.lib.database.building.Building;
```

Comment:
Generally, would it improve readability to separate between SqLiteLoader / Reader and SqLiteWriter?

(the file has already 966 lines)

Reply:
Good idea, I will check this :)